# Define the goal
geometry_msgs/Point posR
geometry_msgs/Point posL
float32 angleR
float32 angleL
float32 duration
---
# Define the result
bool reached_goal
---
# Define a feedback message
float32 progress